src/components/toolbar/import-buttons/hooks/use-schedule-converter.ts

```diff
@@ -53,5 +48,34 @@ export function useScheduleConverter(): UseScheduleConverterOutput {
     scheduleSheet: scheduleSheet,
     setSrcFile: setSrcFile,
     scheduleErrors: scheduleErrors,
+    errorOccurred: errorOccurred,
   };
+
+  function readFileContent(workbook): void {
+    const sheet = workbook.getWorksheet(1);
+    if (sheet.rowCount === 0) {
+      throw new Error("EMPTY_FILE");
```

Comment:
Ja bym użył w tym miejscu `enum`. 
I zminiełbym ten enum co ty zrobiłaś zmieniłbym w taki sposób, że wartości będą takie same jak klucze. 
Czyli:
```typescript
export enum InputFileErrorCode {
  EMPTY_FILE = "EMPTY_FILE",
  NO_CARETAKER_SECTION = "NO_CARETAKER_SECTION",
  NO_NURSE_SECTION = "NO_NURSE_SECTION",
  NO_CHILDREN_SECTION = "NO_CHILDREN_SECTION",
  INVALID_METADATA = "INVALID_METADATA",
  NO_CHILDREN_QUANTITY = "NO_CHILDREN_QUANTITY",
}
```

Reply:
I to było by fajne pod tym względem, że trudniej by było się pomylić (np. zrobić literówkę ) przy wyrzuceniu wyjątku z kodem.